src/components/common/ActionModal.vue

```diff
@@ -17,6 +13,30 @@
           <i class="material-icons">close</i>
         </div>
       </div>
+      <!-- This step stuff should be refactored into something smaller -->
```

Comment:
this is a good start but will restrict all action modals to the same 3 steps (which might be ok for now). it would be worth thinking about how to generalize this for any number of steps.

Reply:
Yeah I was thinking about how to create a `stepGroup` component that takes in an array of strings. I guess that could work but it feels like passing an array of strings is pretty clunky.